### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by grace-mugoiri**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,20 @@
...
+
+- Internet
+
+# Stack used
+
+This project is entirely created using HTML and CSS.
+
+
+
+# Github pages
```
Comment (azdrenymeri) [inline README.md]: Just change it to ` #Live Version` 😃

File: assets/css/main.css [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+}
+
+#advert{
+  width: 60%;
+  height:619px;
+  background: #bf392f;
+  border-radius: 5px;
+  margin-left: 7%;
+}
+.col-1-3{
```
Comment (azdrenymeri) [inline assets/css/main.css:105]: This `.col-1-3` selector is declared in line `61` also

File: index.html [hunk 3]
```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Smash Magazine</title>
+    <link rel="stylesheet" href="assets/css/main.css">
+</head>
+<body>
+    <div class="grid">
```
Comment (azdrenymeri) [inline index.html]: change this to a `nav` tag

File: index.html [hunk 4]
```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html>
```
Comment (azdrenymeri) [inline index.html]: add `lang="en"` attribute to the html tag

**Review by azdrenymeri — APPROVED** [review]
## Well Done 👍 
### Just make those small fixes
No further instructions from my side

**Comment by grace-mugoiri** [comment]
hey, i fixed the errors, just one thing i didnt get this one ==>Just change it to #Live Version 😃, 
and then should i ask for another code review or merge it directly?

**Pull request merged**